.circleci/config.yml

```diff
@@ -75,7 +86,8 @@ references:
     ENVIRONMENT: release
     HELM_REPOSITORY: codacy-incubator
     CODACY_URL: http://release.k8s.dev.codacy.org
-    HELM_INSTALL_EXTRA_FLAGS: --set global.github.app.name=codacy-k8s-selfhost-release --set global.github.app.id="${GITHUB_APP_ID_RELEASE}" --set global.github.app.privateKey="${GITHUB_PRIVATE_KEY_RELEASE}" --set global.github.clientId=${GITHUB_CLIENTID_RELEASE} --set global.github.clientSecret=${GITHUB_CLIENTSECRET_RELEASE} --set global.githubEnterprise.app.name=codacy-k8s-selfhost-release --set global.githubEnterprise.app.id="${GITHUB_ENTERPRISE_APP_ID_RELEASE}" --set global.githubEnterprise.app.privateKey="${GITHUB_ENTERPRISE_PRIVATE_KEY_RELEASE}" --set global.githubEnterprise.clientId=${GITHUB_ENTERPRISE_CLIENTID_RELEASE} --set global.githubEnterprise.clientSecret=${GITHUB_ENTERPRISE_CLIENTSECRET_RELEASE} --set global.bitbucket.key="${BITBUCKET_KEY_RELEASE}" --set global.bitbucket.secret="${BITBUCKET_SECRET_RELEASE}" --set global.bitbucketEnterprise.consumerKey="${BITBUCKET_ENTERPRISE_CONSUMER_KEY_RELEASE}" --set global.bitbucketEnterprise.consumerPublicKey="${BITBUCKET_ENTERPRISE_CONSUMER_PUBLIC_KEY_RELEASE}" --set global.bitbucketEnterprise.consumerPrivateKey="${BITBUCKET_ENTERPRISE_CONSUMER_PRIVATE_KEY_RELEASE}" --set global.gitlab.clientId="${GITLAB_CLIENTID_RELEASE}" --set global.gitlab.clientSecret="${GITLAB_CLIENTSECRET_RELEASE}" --set global.gitlabEnterprise.clientId="${GITLABENTERPRISE_CLIENTID_RELEASE}" --set global.gitlabEnterprise.clientSecret="${GITLABENTERPRISE_CLIENTSECRET_RELEASE}"
+    GITHUB_APP_NAME_RELEASE: "codacy-k8s-selfhost-release"
+    GITHUB_ENTERPRISE_APP_NAME_RELEASE: "codacy-k8s-selfhost-release"
```

Comment:
Should this be codacy-k8s-selfhost**ed**-release?

Reply:
Yes, it is inconsistent, but if you take a look at the diff that's what's being used for those github apps. Can't change without braking stuff.

Reply:
Isn't this the bug machado found on friday https://github.com/codacy/chart/pull/436 ?

Reply:
Thanks for the heads up! 👍